api/HttpClient.ts

```diff
@@ -179,7 +180,14 @@ export class HttpClient implements ifm.IHttpClient {
         var socket;
 
         if (objs) {
-            reqData = JSON.stringify(objs, null, 2);
+            var urlEncFormContentType: boolean = options.headers["Content-Type"] && options.headers["Content-Type"].indexOf("application/x-www-form-urlencoded") != -1;
```

Comment:
This will be a moot point.  It's changing to be a Buffer.  It's up to the consumer to json'ify or not.  This is an http client and it should not be biased toward JSON.  However, the layer about it (rest client) will be.